test/PrizePool.t.sol

```diff
@@ -644,7 +644,32 @@ contract PrizePoolTest is Test {
         assertEq(prizePool.nextDrawStartsAt(), lastCompletedDrawStartedAt+drawPeriodSeconds);
     }
 
+    function testNextDrawIncludesMissedDraws() public {
+        assertEq(prizePool.getNextDrawId(), 1);
+        vm.warp(lastCompletedDrawStartedAt + drawPeriodSeconds * 2);
+        assertEq(prizePool.nextDrawStartsAt(), lastCompletedDrawStartedAt + drawPeriodSeconds);
+        assertEq(prizePool.nextDrawEndsAt(), lastCompletedDrawStartedAt + drawPeriodSeconds * 2);
+        completeAndStartNextDraw(winningRandomNumber);
+        assertEq(prizePool.getNextDrawId(), 2);
+    }
+
+    function testNextDrawIncludesMissedDraws_middleOfDraw() public {
+        assertEq(prizePool.getNextDrawId(), 1);
+        vm.warp(lastCompletedDrawStartedAt + (drawPeriodSeconds * 5) / 2);
+        assertEq(prizePool.nextDrawStartsAt(), lastCompletedDrawStartedAt + drawPeriodSeconds);
+        assertEq(prizePool.nextDrawEndsAt(), lastCompletedDrawStartedAt + drawPeriodSeconds * 2);
+        completeAndStartNextDraw(winningRandomNumber);
+        assertEq(prizePool.getNextDrawId(), 2);
+    }
 
+    function testNextDrawIncludesMissedDraws_2Draws() public {
+        assertEq(prizePool.getNextDrawId(), 1);
+        vm.warp(lastCompletedDrawStartedAt + drawPeriodSeconds * 3);
+        assertEq(prizePool.nextDrawStartsAt(), lastCompletedDrawStartedAt + drawPeriodSeconds * 2);
+        assertEq(prizePool.nextDrawEndsAt(), lastCompletedDrawStartedAt + drawPeriodSeconds * 3);
+        completeAndStartNextDraw(winningRandomNumber);
+        assertEq(prizePool.getNextDrawId(), 2);
```

Comment:
These are good tests; but it seems like they're all for the initial state of no draw.  Maybe check the code coverage results? I feel like some branch is being missed

might be worth adding a few test cases for when a few draws have completed already